vapi/namespace/internal/internal.go

```diff
@@ -22,6 +22,9 @@ const (
 	NamespaceDistributedSwitchCompatibility = "/api/vcenter/namespace-management/distributed-switch-compatibility"
 	NamespaceEdgeClusterCompatibility       = "/api/vcenter/namespace-management/edge-cluster-compatibility"
 	SupervisorServicesPath                  = "/api/vcenter/namespace-management/supervisor-services"
+
+	NamespacesPath = "/api/vcenter/namespaces/instances"
```

Comment:
Should we update the copyright of the changed files with `2020-2024` (in that one)?

Reply:
Good point, header updated